stencil-library/src/components/business-info/business-info.tsx

```diff
@@ -0,0 +1,161 @@
+import { Component, Host, h, Prop, State, Listen, Method } from '@stencil/core';
+import { ValidationError } from 'yup';
+import { Api } from '../../api';
+import BusinessInfoSchema, { BusinessStructureOptions, BusinessTypeOptions, IBusinessInfo } from './business-info-schema';
+
+
+/**
+ * @exportedPart label: Label for inputs
+ * @exportedPart input: The input fields
+ * @exportedPart input-invalid: Invalid state for inputfs
+ */
+@Component({
+  tag: 'justifi-business-info',
+  styleUrl: 'business-info.scss',
+  shadow: true,
+})
+export class BusinessInfo {
+  @Prop() authToken: string;
+  @Prop() businessId?: string;
+  @State() businessInfoPrefillData: any;
```

Comment:
I think this will make more sense when the component is more complete, but basically we will need a way to know later on whether the form needs to do a `post` or a `patch` request on submit, based on whether it loaded in existing data or not. We could also use this to persist fetched business data so that it can be used to reset the form during an edit

Reply:
That's fair